docs/Emerging-Issues.md

```diff
@@ -11,7 +11,7 @@ This page lists emerging issues for Exchange On-Premises deployments, possible r
 
 **Issue** |**Possible reason**| **Workaround/Solution**
 -|-|-
-After installing [March 2022 Security Update For Exchange Server 2013, 2016, 2019](https://techcommunity.microsoft.com/t5/exchange-team-blog/released-march-2022-exchange-server-security-updates/ba-p/3247586), the Microsoft Exchange Service Host service may crash repeatedly with Event ID 7031 in system log and Event ID 4999 in application log. <BR><BR>  Event ID 4999 <BR> Watson report about to be sent for process id: 4564, with parameters: E12IIS, c-RTL-AMD64, 15.01.2375.024, M.Exchange.ServiceHost, M.Exchange.Diagnostics, M.E.D.ChainedSerializationBinder.LoadType, M.E.Diagnostics.BlockedDeserializeTypeException, c0e9-dumptidset, 15.01.2375.024.|The issue can occur if there are any expired certificates present on or any certificates nearing expiry on the server| **Update 3/16/2022** <BR><BR> Follow the steps from [KB 5013118](https://support.microsoft.com/kb/5013118) to resolve the issue | NA | 1) Run the Get-MailboxDatabaseCopyStatus command from Exchange 2016/2019 servers <BR> 2) For Exchange Admin Center issue, make sure the mailbox of admin account is on Exchange 2016/2019 servers.
+After installing [March 2022 Security Update For Exchange Server 2013, 2016, 2019](https://techcommunity.microsoft.com/t5/exchange-team-blog/released-march-2022-exchange-server-security-updates/ba-p/3247586), the Microsoft Exchange Service Host service may crash repeatedly with Event ID 7031 in system log and Event ID 4999 in application log. <BR><BR>  Event ID 4999 <BR> Watson report about to be sent for process id: 4564, with parameters: E12IIS, c-RTL-AMD64, 15.01.2375.024, M.Exchange.ServiceHost, M.Exchange.Diagnostics, M.E.D.ChainedSerializationBinder.LoadType, M.E.Diagnostics.BlockedDeserializeTypeException, c0e9-dumptidset, 15.01.2375.024.|The issue can occur if there are any expired certificates present on or any certificates nearing expiry on the server| **Update 5/11/2022** <BR><BR> Please install [March 2022 security update](https://techcommunity.microsoft.com/t5/exchange-team-blog/released-may-2022-exchange-server-security-updates/bc-p/3356108#M32888) to fix the issue  <BR><BR> **Update 3/16/2022** <BR><BR> Follow the steps from [KB 5013118](https://support.microsoft.com/kb/5013118) to resolve the issue | NA | 1) Run the Get-MailboxDatabaseCopyStatus command from Exchange 2016/2019 servers <BR> 2) For Exchange Admin Center issue, make sure the mailbox of admin account is on Exchange 2016/2019 servers.
```

Comment:
Please install [March 2022 security update]

I think you mean the May 2022 security update, right?

Reply:
My bad! yes of course, it's May 2022